scripts/setup/restore-backup

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+DATABASE_NAME="zulip"
+files_path="/home/zulip/uploads"
+settings_path="/etc/zulip"
```

Comment:
This only works in production; we should probably figure out how to make this work in development too.